code/modules/mob/inventory.dm

```diff
@@ -129,7 +129,8 @@
 	if(!canUnEquip(I))
 		return
 	SEND_SIGNAL(src, COMSIG_CLOTH_DROPPED, I)
```

Comment:
What about this signal, it also uses `I`, is it s problem too?

Reply:
passing arguments as null is not a problem , but sending our signal to  null is.